exasol_integration_test_docker_environment/test/test_hash_temp_dir.py

```diff
@@ -206,12 +216,161 @@ def test_directory_file_names_not_equal_to_dir_names(self):
                                     use_relative_paths=True,
                                     hash_directory_names=True,
                                     hash_file_names=False)
-        hash1_content_only = hasher_content_only.hash([self.test_dir1])
-        hash2_with_paths = hasher_with_paths.hash([self.test_dir2])
+        hash1_content_only = hasher_content_only.hash([simple_path_mapping(self.test_dir1)])
+        hash2_with_paths = hasher_with_paths.hash([simple_path_mapping(self.test_dir2)])
         ascii_hash1_content_only = base64.b32encode(hash1_content_only).decode("ASCII")
         ascii_hash2_with_paths = base64.b32encode(hash2_with_paths).decode("ASCII")
         self.assertNotEqual(ascii_hash1_content_only, ascii_hash2_with_paths)
 
+    def test_file_name_with_relative_path(self):
+        """
+        Test that hashing of same files in different paths gives same result
+        """
+        hasher_content_only = \
+            FileDirectoryListHasher(followlinks=True,
+                                    hashfunc="sha256",
+                                    hash_file_names=True,
+                                    hash_directory_names=True,
+                                    hash_permissions=True,
+                                    use_relative_paths=True)
+        test_file1 = f"{self.test_dir1}/test.txt"
+        with open(test_file1, "w") as f:
+            f.write("test")
+            hash1_content_only = hasher_content_only.hash([PathMapping("test.txt", test_file1)])
+
+        test_file2 = f"{self.test_dir2}/test.txt"
+        with open(test_file2, "w") as f:
+            f.write("test")
+            hash2_content_only = hasher_content_only.hash([PathMapping("test.txt", test_file2)])
+
+        ascii_hash1_content_only = base64.b32encode(hash1_content_only).decode("ASCII")
+        ascii_hash2_content_only = base64.b32encode(hash2_content_only).decode("ASCII")
+        self.assertEqual(ascii_hash1_content_only, ascii_hash2_content_only)
+
+    def test_file_name_with_relative_path_in_same_sub_path(self):
```

Comment:
You are modifying the test directories, this can interfere with other tests. We can't assume test execution order

Reply:
Fixed